components/dialog/predefined.md

```diff
@@ -10,7 +10,7 @@ position: 2
 
 # Predefined Dialogs - Alert, Confirm, Prompt
 
-Telerik UI for Blazor provides styled substitutes to the standard confirm, alert and prompt dialogs. They match the Theme of the components to make it obvious to the user that the modal dialog is coming from your application.
+Telerik UI for Blazor provides styled substitutes to the standard confirm, alert and prompt dialogs. The Blazor popup messages match the Theme of the components to make it obvious to the user that the modal dialog is coming from your application.
 
 To use these dialogs, receive a cascading parameter of type `Telerik.Blazor.DialogFactory`. It exposes the methods you can use in your method calls.
```

Comment:
When calling one thing different names, we introduce ambiguity, so make sure the reader is still able to understand what we're referring to.
```suggestion
To use these dialogs (popup messages), receive a cascading parameter of type `Telerik.Blazor.DialogFactory`. It exposes the methods you can use in your method calls.
```